content/docs/pipelines/sdk/dsl-recursion.md

```diff
@@ -0,0 +1,91 @@
++++
+title = "DSL Recursion"
+description = "Author a recursive function in DSL."
```

Comment:
We've followed the pattern of having no full stop (period) at the end of the short description on each page, as the description is often not a full sentence.

Please remove the full stop. :)

Reply:
Sure.